std_msgs/Header header

uint32 round
geometry_msgs/Pose pose
geometry_msgs/Pose odom
geometry_msgs/Twist autoware_twist
geometry_msgs/Twist ypspur_twist
float32 brake
float32 accel
uint32 shift
uint32 obstacle_id
uint32 detected_flag
uint32 pedestrian_flag
